cmd/cluster/ssh/key.go

```diff
@@ -0,0 +1,157 @@
+package ssh
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/openshift/osdctl/pkg/k8s"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// sshSecretName defines the name of the ssh secret in each hive namespace
+	sshSecretName = "ssh"
+	// privateKeyFilename defines the map key used to identify the private ssh key in the hive "ssh" secret's data
+	privateKeyFilename = "ssh-privatekey"
+)
+
+func NewCmdKey() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "key [cluster identifier]",
+		Short: "Retrieve a cluster's SSH key from Hive",
+		Long:  "Retrieve a cluster's SSH key from Hive. If a cluster identifier (internal ID, UUID, name, etc) is provided, then the key retrieved will be for that cluster. If no identifier is provided, then the key for the cluster backplane is currently logged into will be used instead.",
```

Comment:
Can we add the specific use case for this command to its description? I'm worried we might accidentally over-use this when we don't have to. This should only be used as an absolute last resort.